### PR TITLE
added 5.1 status reporter support

### PR DESCRIPTION
**Pull request by AdarShaked**
updated image to 5.1 and added postgres_exporeter to db config

removed credentials

### CODE REVIEW

File: scripts/cluster-install/environment_ids.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
-{jump-host: 0e77f615-c682-4af2-ae36-78b18f4a943a, load_balancer: c6bfebb2-a902-4811-ab99-76ff9c525e27,
-  manager-1: 4aa9eb73-12c0-4fd8-bc88-f70e1c0d77ae, manager-2: c1afbd05-35db-4619-bb78-622774dd02d5,
-  manager-3: 6858ca1d-1cf3-4f92-8689-57dedc343171, postgresql-1: b86128ff-e0a5-4526-932c-a35977b254dd,
-  postgresql-2: 31aaeb23-958b-41db-bf5d-09b5dad20c90, postgresql-3: 5eb4cb9f-164d-448f-8d63-6b7251a6082b,
-  rabbitmq-1: 6c8375d6-c990-4385-af12-0c05586d66ef, rabbitmq-2: 5bd1e89e-ef63-42c2-b849-45dd251c21de,
-  rabbitmq-3: 92fdf5e4-42bf-4062-9418-3b505bc8ad49, sec_group: 53242d06-3e29-465d-ae9f-2e4c61018d2c}
+{jump-host: 9659e4e0-1089-4c74-ad3a-187870516eb5, load_balancer: 0a0364c0-9d01-44f6-a284-a0c259f28b1d,
```
Comment (YoniItzhak) [inline scripts/cluster-install/environment_ids.yaml:1]: This is not relevant as everyone has a different environment_ids file.

Comment (YoniItzhak) [inline scripts/cluster-install/environment_ids.yaml:1]: This is not relevant as everyone has a different environment_ids file. Probably just slipped inside the PR :)

File: scripts/cluster-install/postgresql_config.yaml [hunk 2]
```diff
@@ -30,11 +30,18 @@ postgresql_server:
...
+  credentials:
+    username: 'monitoringusername'
+    password: 'longyeteasytorememberstringasapassword'
+
+  cert_path:
+  key_path:
+  ca_path:
   postgres_exporter:
-    # `password` is a placeholder and will be updated during config file rendering, based on postgresql_server.postgres_password
+#    # `password` is a placeholder and will be updated during config file rendering, based on postgresql_server.postgres_password
```
Comment (YoniItzhak) [inline scripts/cluster-install/postgresql_config.yaml:41]: The "#" is no relevant here? Probably also slipped inside

File: scripts/cluster-install/environment_ids.yaml (hunk 1 above)
Comment (AdarShaked) [reply scripts/cluster-install/environment_ids.yaml:1]: yes its mistake. i will add this file  to .gitignore

**Pull request merged**